spec/03-plugins/03-http-log/01-log_spec.lua

```diff
@@ -269,4 +312,67 @@ for _, strategy in helpers.each_strategy() do
       end, 10)
     end)
   end)
-end
+
+  describe("Plugin: http-log (log) [#" .. strategy .. "]", function()
+    local proxy_client
+
+    setup(function()
```

Comment:
Note: we now use `lazy_setup` (see the recent commits on master)

Reply:
Added lazy_setup/lazy_tear_down, I am all for lazy :D .